### ISSUE TITLE
internal error: package "<...>" without types was imported from "command-line-arguments"

### ISSUE THREAD

**Issue by Integralist** [securego/gosec#797]
### Summary 

Since upgrading to golang 1.18 I'm seeing the following error:

```
internal error: package "<...>" without types was imported from "command-line-arguments"
```

Where `<...>` can be replaced for any package.

### Steps to reproduce the behavior

Here are two examples...

1. fmt
2. embed

**fmt**:

```go
package main

import (
	"fmt"
)

func main() {
	fmt.Println("example")
}
```

**embed**

Create an `example.txt` with some content in it.

```go
package main

import (
	_ "embed"
	"fmt"
)

//go:embed example.txt
var metadata []byte

func main() {
	fmt.Println(string(metadata))
}
```

### gosec version

Unfortunately I'm unable to provide this information because `gosec --version` returns empty values...

```
$ gosec --version
Version:
Git tag:
Build date:
```

### Go version (output of 'go version')

```
go version go1.18 darwin/amd64
```

### Operating system / Environment

macOS Monterey

### Expected behavior

No errors reported.

### Actual behavior

An internal error is reported.

**Comment by ccojocar** [comment]
Do you use the lastest release? https://github.com/securego/gosec/releases/tag/v2.11.0 

This was built with Go 1.18.

**Comment by Integralist** [comment]
@ccojocar this is the tricky thing. I've tried both...

```
go install github.com/securego/gosec/cmd/gosec@latest
go install github.com/securego/gosec/cmd/gosec@master
```

And regardless, when I run `gosec --version` I get...

```
Version:
Git tag:
Build date:
```

So I can't actually tell which version I'm getting :-/ 

If I do `which gosec` it's going to `/Users/integralist/go/bin/gosec` which is where I expect the binary to be installed.

**Comment by Integralist** [comment]
I've also tried installing the specific tag revision, but that errors:

```
$ go install github.com/securego/gosec/cmd/gosec@v2.11.0
go: github.com/securego/gosec/cmd/gosec@v2.11.0: github.com/securego/gosec/cmd/gosec@v2.11.0: invalid version: unknown revision cmd/gosec/v2.11.0
```

**Comment by ccojocar** [comment]
The install command is not correct. This is installing the v1 and not v2. You need to run: 
```
go install github.com/securego/gosec/v2/cmd/gosec@latest
```

You can read this in the README file go install github.com/securego/gosec/v2/cmd/gosec@latest. 

You can also download the compiled binary as described above. Both should work.

**Comment by Integralist** [comment]
Oh man. My apologies 🤦🏻 

Thank you for pointing me in the right direction.

I can confirm that it does now indeed run for 1.18 🥳 

But FYI `gosec --version` still reports empty values 🤔 (not a problem for me, but just mentioning in case this is a regression).

Thanks again!

**Comment by ccojocar** [comment]
Great! The version is embedded only in the official released binary. The install command doesn't embed the version. This capability is now available in 1.18, hopefully we will add it soon.
